src/yara.cc

```diff
@@ -222,6 +222,7 @@ void ScannerWrap::Init(Local<Object> exports) {
 	Nan::SetPrototypeMethod(tpl, "configure", Configure);
 	Nan::SetPrototypeMethod(tpl, "scan", Scan);
 	Nan::SetPrototypeMethod(tpl, "getRules", GetRules);
+	Nan::SetPrototypeMethod(tpl, "reconfigureRules", ReconfigureRules);
```

Comment:
From reading the implementation it looks like this will only reconfigure variables right now? If so, might be better to rename it, unless the plan is to extend it.

Reply:
Good point! I'll rename it to reconfigureVariables.